ui/pages/confirm-transaction-base/transaction-alerts/transaction-alerts.test.js

```diff
@@ -1,172 +1,307 @@
 import React from 'react';
```

Comment:
I opted to refactor these tests so that they're simpler and don't require us to recreate the Redux state so much. I can put these changes in another PR if necessary, but I will point the additional check for `isNetworkBusy` regardless.